lib/src/bazelify/build.dart

```diff
@@ -370,3 +378,45 @@ class HtmlEntryPoint {
     assert(htmlFile != null);
   }
 }
+
+/// A `dev_server` definition for one or more ddc web apps.
+class DdcDevServer implements DartBuildRule {
+  @override
+  final String name;
+
+  @override
+  String get package =>
+      throw new UnimplementedError('DdcDevServer doesn\'t need a package');
+
+  /// one or more web applications this server serves
+  final List<DartWebApplication> webApps;
+
+  /// Create a new `dev_server` named [name] executing [webApps].
+  const DdcDevServer({this.name, this.webApps});
+
+  @override
+  String toRule() {
+    var buffer = new StringBuffer();
+    if (webApps.isNotEmpty) {
+      buffer.writeln('# A ddc specific dev_server target which serves '
+          '${webApps.map((app) => app.name).join(', ')}');
+      buffer.writeln('dev_server(\n'
+          '    name = "$name",\n'
+          '    data = [');
+      for (var webApp in webApps) {
+        buffer.writeln('        ":${webApp.ddcBundleName}",');
+      }
+      buffer.writeln('    ],\n'
+          '    script_args = [\n'
+          '        "--package-spec='
+          '${webApps.first.packageSpecName}",');
```

Comment:
maybe add a comment that all webApps will have the same package spec since they share the same dependencies in the pubspec.yaml